libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -194,27 +202,40 @@ class PreviewImageFragment : Fragment() {
     }
 
     private fun setupObservers() {
-        viewModel.uiState.observe(this, Observer { uiStateEvent ->
-            uiStateEvent?.getContentIfNotHandled()?.let { state ->
-                (previewImageViewPager.adapter as PreviewImageAdapter).submitList(state.viewPagerItemsStates)
-                cropActionMenu?.isEnabled = state.editActionsEnabled
-                UiHelpers.updateVisibility(thumbnailsTabLayout, state.thumbnailsTabLayoutVisible)
-            }
+        viewModel.uiState.observe(viewLifecycleOwner, Observer { state ->
+            (previewImageViewPager.adapter as PreviewImageAdapter).submitList(state.viewPagerItemsStates)
+            cropActionMenu?.isEnabled = state.editActionsEnabled
+            UiHelpers.updateVisibility(thumbnailsTabLayout, state.thumbnailsTabLayoutVisible)
         })
 
-        viewModel.loadIntoFile.observe(this, Observer { fileStateEvent ->
+        viewModel.loadIntoFile.observe(viewLifecycleOwner, Observer { fileStateEvent ->
             fileStateEvent?.getContentIfNotHandled()?.let { fileState ->
                 if (fileState is ImageStartLoadingToFileState) {
                     loadIntoFile(fileState.imageUrlAtPosition, fileState.position)
                 }
             }
         })
 
-        viewModel.navigateToCropScreenWithFileInfo.observe(this, Observer { fileInfoEvent ->
+        viewModel.navigateToCropScreenWithFileInfo.observe(viewLifecycleOwner, Observer { fileInfoEvent ->
             fileInfoEvent?.getContentIfNotHandled()?.let { fileInfo ->
                 navigateToCropScreenWithFileInfo(fileInfo)
             }
         })
+
+        findNavController().currentBackStackEntry?.savedStateHandle
+                ?.getLiveData<CropResult>(CropFragment.CROP_RESULT)?.observe(
+                viewLifecycleOwner, Observer { result ->
+            Log.d(PreviewImageFragment::javaClass.name, "result")
```

Comment:
This log slipped through during the pair programming session. I assume we'll want to remove it.

Reply:
Removed here: 557bb3e